galite-core/src/main/kotlin/org/kopi/galite/base/ReflectTool.kt

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.kopi.galite.base
+
+import org.jetbrains.exposed.sql.Table
+import org.kopi.galite.common.Actor
+import org.kopi.galite.form.dsl.*
+import kotlin.reflect.KClass
+import kotlin.reflect.full.isSubclassOf
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.jvm.jvmErasure
+
+/**
+ * Class of reflection tools
+ * Contains methods which access to formBlocks' fields and indices and tables
+ * and @return the element of index requested
```

Comment:
improve this comment to specify that the class access all components defined using Galite DSL like formBlocks' fields, indices, actors, etc.